R/feature_groups.R

```diff
@@ -1352,6 +1352,68 @@ setMethod("calculateTox", "featureGroups", function(fGroups, featureAnn)
 })
 
 
+#' @describeIn featureGroups Obtain the total ion chromatogram/s (TICs) of the analyses (`i`).
+#' @param msLevels Integer vector with the ms levels (i.e., 1 for MS1 and 2 for MS2) 
+#' to obtain TIC traces.
+#' @param retentionRange Range of retention time (in seconds) to collect TIC traces.
+#' Should be a numeric vector with length of two containing the min/max values. 
+#' Set to NULL to ignore.
+#' @aliases getTICs
+#' @export
+setMethod("getTICs", "featureGroups", function(obj, i = NULL, retentionRange = NULL, msLevels = c(1, 2))
```

Comment:
* For consistency with other functions, I would say to rename `msLevels` to `MSLevel`.
* Please make sure to use 4 space indenting.

Reply:
Agree